ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/QueryAnonymizerTest.java

```diff
@@ -13,32 +13,34 @@
 package io.confluent.ksql.test;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
+
 import io.confluent.ksql.engine.rewrite.QueryAnonymizer;
 import io.confluent.ksql.test.QueryTranslationTest.QttTestFile;
 import io.confluent.ksql.test.loader.JsonTestLoader;
 import io.confluent.ksql.test.tools.TestCase;
 import io.confluent.ksql.util.GrammarTokenExporter;
 import java.io.BufferedReader;
 import java.io.FileInputStream;
-import java.io.FileReader;
 import java.io.InputStreamReader;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.Collection;
-import java.util.Scanner;
-import java.util.stream.Collectors;
-import org.approvaltests.Approvals;
+import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 import java.util.stream.Stream;
+import org.approvaltests.Approvals;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
+@RunWith(Enclosed.class)
```

Comment:
thanks for the fix, as per the docs , abstract inner classes are still ignored. But it should work for this case.